RasterPropMonitor/Core/RPMCEvaluators.cs

```diff
@@ -2332,6 +2332,36 @@ internal VariableEvaluator GetEvaluator(string input, out bool cacheable)
                 case "FUNDS":
                     return (string variable, RPMVesselComputer comp) => { return Funding.Instance != null ? Funding.Instance.Funds : 0.0; };
 
+                // CommNet
+                case "COMMNETCONNECTED":
+                    return (string variable, RPMVesselComputer comp) => {
+                        return ((vessel.connection != null) && (vessel.connection.IsConnected)) ? 1.0 : 0.0;
+                    };
+                case "COMMNETSIGNALSTRENGTH":
+                    return (string variable, RPMVesselComputer comp) => {
+                        return (vessel.connection != null) ? (float)vessel.connection.SignalStrength : 0.0;
+                    };
+                case "COMMNETVESSELCONTROLSTATE":
+                    return (string variable, RPMVesselComputer comp) => {
+                        if (vessel.connection == null)
+                            return 0.0;
+                        
+                        switch (vessel.connection.ControlState)
+                        {
+                            case CommNet.VesselControlState.None: return 0.0;
+                            case CommNet.VesselControlState.Probe: return 2.0;
```

Comment:
Do you mean, why not just convert the enum to float? I figured it would be better to not do that since the enum values can be changed, and if that happened it would be a mess.